src/assemble_workflow/dist.py

```diff
@@ -109,6 +110,14 @@ def __build__(self, name: str, dest: str) -> None:
                     zip.write(fn, fn[rootlen:])
 
 
+class DistDeb(Dist):
+
+    def __extract__(self, dest: str) -> None:
+        BundleDeb(self.filename, self.path, self.min_path).extract(dest)
```

Comment:
Add a constructor and initialize a `bundle` property that is reused across these methods.

Reply:
@dblock thanks for reviewing this, I split these files, please have a look again.